### ISSUE TITLE
Svgs not rendering properly

### ISSUE THREAD

**Issue by nboliver** [react-native-community/react-native-svg#1178]
When using a previewer such as https://www.freecodeformat.com/svg-editor.php it renders as expected. When viewing in React Native app, it appears garbled: 

![image](https://user-images.githubusercontent.com/3094055/68179450-c6d1ad80-ff44-11e9-9e6c-84b32ffcaea6.png)

Svg component:
```
<Svg width={20} height={24} viewBox="0 0 20 24" fill="none">
  <Path
    d="M15.892 6.162A6.16 6.16 0 009.73 0a6.16 6.16 0 00-6.162 6.162 6.16 6.16 0 006.162 6.162c3.405 0 6.162-2.789 6.162-6.162zm-10.703 0a4.554 4.554 0 014.54-4.54 4.554 4.554 0 014.541 4.54 4.554 4.554 0 01-4.54 4.54 4.554 4.554 0 01-4.54-4.54zM19.46 24v-.81c0-5.644-4.282-10.055-9.73-10.055C4.28 13.135 0 17.546 0 23.19V24h1.622v-.81c0-4.8 3.502-8.433 8.108-8.433 4.605 0 8.108 3.632 8.108 8.432V24h1.621z"
    fill="#000"
  />
</Svg>
```

## Environment info

Expo: 3.0.10
Library version: react-native-svg@9.5.3

**Comment by msand** [comment]
Can you try with the latest version?

**Comment by nboliver** [comment]
@msand Thanks for a quick reply.
Updated Expo to v3.4.1
Using `expo install react-native-svg` latest version available seems to be 9.5.3
Tried forcing latest version with `yarn add  react-native-svg@~9.13.3` but issue persists.

**Comment by msand** [comment]
This is probably fixed in the latest version, so either eject expo or make a plain react-native project, or wait for expo to update.
